File: 1000-9999/5000/5063.cpp
// TGN
#include <bits/stdc++.h>
#define ll long long 
using namespace std;

int check[31];
int main() {
	ios_base::sync_with_stdio(0);
	cin.tie(0); cout.tie(0);
	int q;
	cin >> q;
	while(q--){
		int r, e, c;
		cin >> r >> e >> c;
		if (e - c > r)
			cout << "advertise" << '\n';
		else if (e - c == r)
			cout << "does not matter" << '\n';
		else
			cout << "do not advertise" << '\n';
	}
}

File: 1000-9999/5000/5063.py
# TGN
N = int(input())
for i in range(N):
    r, e, c = map(int, input().split())
    a = r + c
    if a < e:
        print("advertise")
    elif a == e:
        print("does not matter")
    else:
        print("do not advertise")
